andino_gazebo/worlds/empty.world

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" ?>
+<sdf version="1.6">
+  <world name="default">
+    <plugin name="gazebo_ros_state" filename="libgazebo_ros_state.so">
+      <ros>
+      </ros>
+
+      <update_rate>1.0</update_rate>
```

Comment:
Just double checking, is it ok 1hz for your application? @elector102 @agalbachicar

Reply:
Thanks! I will increase that. I think 10 hz for physics things are enough.

Reply:
Fixed!

Reply:
Note that this is just update rate of the plugin. If it happen you want to modify the physics's rate you need to add other tags.

Reply:
10Hz is fine. Thanks for catching it.